### PR TITLE
Warped biome For Enderman hunting

### PR DESCRIPTION
**Pull request by JamesGreen31**
This is the code that will find a warped biome for enderman hunting. This is **just** the code for getting the biome. You will have to implement it in BeatMinecraft2 on your own.

_I don't keep it loaded son, you'll have to find ammo as you go._

### CODE REVIEW

File: src/main/java/adris/altoclef/tasks/movement/LocateWarpedBiomeTask.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
+package adris.altoclef.tasks.movement;
```
Comment (JamesGreen31) [inline src/main/java/adris/altoclef/tasks/movement/LocateWarpedBiomeTask.java:1]: Dis all good

File: .github/ISSUE_TEMPLATE/feature-improvement-request.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 name: Feature/Improvement request
 about: Suggest an enhancement
-title: ''
+title: '[FEATURE NAME]'
```
Comment (JamesGreen31) [inline .github/ISSUE_TEMPLATE/feature-improvement-request.md:4]: I don't know why it pushed these changes. You may have to revoke these in particular

File: .github/ISSUE_TEMPLATE/project_advancement.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+name: Project Advancement
+about: Create a task for a project
+title: ''
+labels: 'Available'
+assignees: ''
+
```
Comment (JamesGreen31) [inline .github/ISSUE_TEMPLATE/project_advancement.md:7]: same with this. I didn't know it would push these. How do I fix that?

**Comment by adrisj7** [comment]
[We already did this by looking for Nylium](https://github.com/gaucho-matrero/altoclef/blob/932dda53be7917a904ea6a1419e7b3dbcdad34d1/src/main/java/adris/altoclef/tasks/speedrun/BeatMinecraft2Task.java#L616), thanks for helping out though

**Comment by JamesGreen31** [comment]
No worries. I pushed this right around the same time that you pushed yours. Figured it would help to have a different version to look at in case of any bugs. Glad we got this feature in :) 

I would add a setting in settings though for how altoclef will get pearls. It should choose between trading and hunting in warped biome. @adrisj7

**Pull request closed without merging**